Ubuntu-14.04/cms/wordpressnginx.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/sh
+#
+# WordPress Setup Script
+#
+# This script will install and configure WordPress on
+# an Ubuntu 14.04 droplet
+export DEBIAN_FRONTEND=noninteractive;
+
+# Generate root and wordpress mysql passwords
+rootmysqlpass=`dd if=/dev/urandom bs=1 count=32 2>/dev/null | base64 -w 0 | rev | cut -b 2- | rev`
+wpmysqlpass=`dd if=/dev/urandom bs=1 count=32 2>/dev/null | base64 -w 0 | rev | cut -b 2- | rev`
+# Write passwords to file
+echo "Root MySQL Password: $rootmysqlpass" > /root/passwords.txt;
+echo "Wordpress MySQL Password: $wpmysqlpass" >> /root/passwords.txt;
+# Update Ubuntu
+apt-get update;
+apt-get -y upgrade;
+# Install Nginx/MySQL
+sudo apt-get install php5-fpm php5-mysql mysql-server mysql-client unzip;
+echo "deb http://ppa.launchpad.net/nginx/stable/ubuntu $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/nginx-stable.list
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys C300EE8C
+sudo apt-get update
+sudo apt-get install nginx
```

Comment:
The script fails to install nginx for me when used in user-data. It hangs here waiting for user input:

```
The following extra packages will be installed:
  fontconfig-config fonts-dejavu-core libfontconfig1 libgd3 libjbig0
  libjpeg-turbo8 libjpeg8 libtiff5 libvpx1 libxpm4 libxslt1.1 nginx-common
  nginx-full
Suggested packages:
  libgd-tools fcgiwrap nginx-doc ssl-cert
The following NEW packages will be installed:
  fontconfig-config fonts-dejavu-core libfontconfig1 libgd3 libjbig0
  libjpeg-turbo8 libjpeg8 libtiff5 libvpx1 libxpm4 libxslt1.1 nginx
  nginx-common nginx-full
0 upgraded, 14 newly installed, 0 to remove and 3 not upgraded.
Need to get 2835 kB of archives.
After this operation, 9273 kB of additional disk space will be used.
Do you want to continue? [Y/n] Abort.
```